packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Create an Identity Provider in Okta
+---
+To connect your org to the Identity Provider, add and configure that Identity Provider in Okta.
+
+1. From the Developer Console, hover over **Users** and then select **Social & Identity Providers** from the menu that appears. If you are using the Classic UI, hover over **Security** and then select **Identity Providers**.
+
+2. Select **Add Identity Provider** and then select the appropriate Identity Provider.
```

Comment:
@ZoeMonosson-okta identified that:

1. If the developer starts in the console...
2. And follows the link from the `Add an Identity Provider` modal to this guide...
3. They will already have done the first two instructions here.

Not sure how much of an issue it is, but the developer may have to think about whether steps 1 + 2 still apply for them, if the first two instructions here still apply for them, etc.

Reply:
@alexdahl-okta - Yes, she did point that out and I have a JIRA ticket out to correct the URL so that it takes them directly to the appropriate page rather than the where it takes them currently. So, they won't be confused any longer, they will arrive on the right page for the dialog they are in.